packages/test/test-end-to-end-tests/src/test/compression.spec.ts

```diff
@@ -102,12 +102,21 @@ describeInstallVersions(
 )("Op Compression self-healing with old loader", (getProvider) =>
 	compressionSuite(async () => {
 		const provider = getProvider();
+		// Ensure support for endpoint names for r11s driver. ODSP might need similar help at some point if we have
+		// scenarios that run into issues otherwise.
+		const driverConfig =
```

Comment:
would it be better to just have drivers, or the provider expose the whole driver config, and directly pass that through?

Reply:
Would be nice and cover both r11s and odsp, yeah. I'll create an ADO task since we don't seem to have an urgent need for it (AFAICT I'm the only one sometimes running e2e tests manually against "unusual" endpoints and thus running into this 😄).